test/Libraries/RevitNodesTests/RevitNodesTests.csproj

```diff
@@ -39,6 +39,11 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="DSCoreNodes, Version=1.0.1.1523, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>../../../../Dynamo/bin/AnyCPU/Debug/DSCoreNodes.dll</HintPath>
```

Comment:
I believe there is a configuration parameter in the config.cs props file that defines a DynamoAPI path you can use here.